calculator.py

```diff
@@ -1,67 +1,79 @@
-def Welcome():
+def main():
```

Comment:
Is it best practice to call functions before the program? or at the end.

Reply:
At the end. Your code looks cleaner that way. You use function to break your code and make it simpler